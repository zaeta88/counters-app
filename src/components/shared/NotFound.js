import React from 'react';
import clouds from '../../assets/img/clouds.svg';

const Notfound = () => (
  <div className="card">
    <div className="card-body" style={{ marginTop: '5%' }}>
      <center>
        <img id="error-img" src={clouds} alt="Error" style={{ width: '5%' }} />
      </center>
      <center style={{
        lineHeight: '18px',
        fontSize: '24px',
        fontWeight: '200',
        marginTop: '8px'
      }}
      >
        <p>This page doesn't exist.</p>
        <p>Verify that you have correctly written the link you want.</p>
      </center>
    </div>
  </div>
);
export default Notfound;
