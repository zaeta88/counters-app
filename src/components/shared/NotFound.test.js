import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';


describe('React.Component NotFound', () => {
  it('should render correctly', () => {
    const component = shallow(<NotFound />);
    expect(component).toMatchSnapshot();
  });
});
