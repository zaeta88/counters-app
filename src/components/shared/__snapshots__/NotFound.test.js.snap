// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React.Component NotFound should render correctly 1`] = `ShallowWrapper {}`;
